*** apsrev.bst	Fri Aug  3 16:40:00 2001
--- apsrev-title.bst	Sat Apr  3 00:35:28 2004
***************
*** 1,3 ****
--- 1,15 ----
+ %% This is file `apsrev-title.bst',
+ %% Copyright (C) 2004 Kengo Ichiki <ichiki@mailaps.org>
+ % $Id: apsrev-title.bst,v 1.1 2004/04/03 05:35:11 ichiki Exp $
+ %
+ % This work may be distributed and/or modified under the
+ % conditions of the LaTeX Project Public License, either version 1.3
+ % of this license or (at your option) any later version.
+ % The latest version of this license is in
+ %   http://www.latex-project.org/lppl.txt
+ % and version 1.3 or later is part of all distributions of LaTeX
+ % version 2003/12/01 or later.
+ %
  %%
  %% This is file `apsrev.bst',
  %% generated with the docstrip utility.
***************
*** 1283,1299 ****
    if$
  }
  
  FUNCTION {format.title}
! { title
!   duplicate$ empty$
!     { skip$ }
!     { "t" change.case$ }
!   if$
!   "title" bibinfo.check
!   duplicate$ empty$ 'skip$
!     {
!       emphasize
!     }
    if$
  }
  
--- 1295,1317 ----
    if$
  }
  
+ %FUNCTION {format.title}
+ %{ title
+ %  duplicate$ empty$
+ %    { skip$ }
+ %    { "t" change.case$ }
+ %  if$
+ %  "title" bibinfo.check
+ %  duplicate$ empty$ 'skip$
+ %    {
+ %      emphasize
+ %    }
+ %  if$
+ %}
  FUNCTION {format.title}
! { title empty$
!     { "" }
!     { "``" title "t" change.case$ * ",''" * }
    if$
  }
  
***************
*** 1796,1801 ****
--- 1814,1821 ----
  FUNCTION {article}
  { output.bibitem
    format.authors "author" output.check
+   format.title "title" output.check
+   add.blank
    author format.key output
    crossref missing$
      {
